Rules/ProvideCommentHelp.cs

```diff
@@ -47,10 +47,17 @@ public override AstVisitAction VisitFunctionDefinition(FunctionDefinitionAst fun
                 return AstVisitAction.SkipChildren;
             }
 
-            if (funcAst.GetHelpContent() == null)
+            if (!string.Equals(funcAst.Name, "Get-TargetResource", StringComparison.OrdinalIgnoreCase) && !string.Equals(funcAst.Name, "Set-TargetResource", StringComparison.OrdinalIgnoreCase) &&
```

Comment:
Can you check with the DSC team if they support aliasing these functions? (Set-Alias).